Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:16.14-alpine AS build
+
+WORKDIR /adcp-ui
+RUN npm i -g npm@8.5.0
+
+# Install base dependencies
+COPY . .
+RUN npm install
+
+# RUN Build
+RUN npm run build
+
+
+##################################################################################################
+
+
+FROM nginx:1.18-alpine AS runtime
+
+WORKDIR /app/adcp-ui/build
+
+COPY --from=build /adcp-ui/build .
+
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
How are we passing that workdir to nginx configuration to use for hosting content?

Reply:
As we're not using helm for this I feel this feel should be removed.
Thoughts?

Reply:
I'd switch the docker image to using node whatever and just `npm run serve` or something.

Reply:
Updated